docs/security/authentication/active-directory/index.md

```diff
@@ -44,21 +44,96 @@ Octopus is built on top of HTTP.sys, the same kernel driver that IIS is built on
+
+### **Configuring Kerberos Authentication for Active Directory** {#ActiveDirectoryAuthentication-ConfiguringKerberos}
+
+The following configuration is required for Kerberos authentication
+- A valid Service Principal Name (SPN) for the `HTTP` service class for each Octopus host NETBIOS name. If you are accessing your Host via it's FQDN then you will need to also add a FQDN also for the `HTTP` service class. (Please Note: Whether you've configured your Octopus host to use `HTTP` or `HTTPS`, you will only need to set a `HTTP` SPN.)
+- Included FQDNs of all Octopus Deploy Hosts and Octopus clusters within your trusted sites or Intranet zones
+- Client Machines configured to allow auto logon with current user name and password
 
-```powershell
-C:\> setspn.exe -S HTTP/od.mydomain.local winserv2019
+
+**SPN Configuration**
+
+Set a `HTTP` service class SPN for the NETBIOS name and FQDN of your OD hosts. For example, if you are hosting `od.mydomain.local` from server `octoserver1` you would require the following registered service principal names for your server.
 ```
+HTTP/od
+HTTP/od.mydomain.local
+```
+These can be registered by running the following commands in a elevated command prompt or powershell session.
+
+```
+setspn.exe -S HTTP/od octoserver1
+setspn.exe -S HTTP/od.mydomain.local octoserver1 
+```
+:::note
+Remember if you are running a HA cluster, you will need to add additional SPN entries for each host you are using.  Load Balancer/cluster urls are not required to be included as an SPN entry.
+:::
 
 For more information about configuration of SPNs [please see this microsoft support article](https://support.microsoft.com/en-us/help/929650/how-to-use-spns-when-you-configure-web-applications-that-are-hosted-on).
 
+**Internet Security Configuration - Adding Octopus to the Trusted Zone**
+
+The aim here is to make sure the current users logon credentials are able to be sent through to Octopus and be authenticated against the SPNs. It is important to remember that a URI is considered to be in the "Internet Zone" whenever it contains a ".". 
+
+```Internet Zone
+http://host.local
+http://192.168.x.x
+http://127.0.0.1
+http://octopus.yourdomain.com
+http://clusterurl.yourdomain.com
+
+Intranet Zone
+http://host
+http://local
+```
+
+Accessing a host via the NETBIOS name will usually mean that the site is considered part of the "Intranet" zone unless the NETBIOS name has been add to "Trusted Sites" list. (More detail [here](https://support.microsoft.com/en-au/help/303650/intranet-site-is-identified-as-an-internet-site-when-you-use-an-fqdn-o)) So although there is a number of ways this can be configured, we recommend the following way of including all URIs that are used to access Octopus, to be added to the "Trusted Sites" zone.
+
+This can be done in several ways, including via Group Policy, scripting or via [browsers settings menu](https://www.computerhope.com/issues/ch001952.htm)  
```

Comment:
```suggestion
This can be done in several ways, including via Group Policy, scripting or via [browsers settings menu](https://www.computerhope.com/issues/ch001952.htm).  
```